torch/csrc/jit/serialization/python_print.cpp

```diff
@@ -751,15 +751,28 @@ struct PythonPrintImpl {
       if (version_entry != get_operator_version_map().end()) {
         const auto& entry = version_entry->second;
         // TODO (tugsuu) move this calculation into a seperate step.
+        uint64_t current_version = entry[entry.size() - 1].bumped_at_version;
+
+        // Make sure this behaviour matches with the historic version
+        // calculation
+        // TODO (tugsuu) this needs to be removed once we think it is safe.
+        uint64_t historic_version = get_min_version_for_kind(node->kind());
+
+        if (historic_version != 0) {
```

Comment:
Could you also add a flag to revert to old behavior? In this error message, let's also let user know that they can use the flag to make it work.

Reply:
Maybe not a flag, since it is harder to use in a notebook. Maybe some new private API called `torch.use_old_upgrader_version`. Up to you.

Reply:
Are you saying we should deprecate the ENABLE_UPGRADERS macro flag and let user flip it through API? Or are you thinking about adding another flag? If latter, I am afraid this logic is getting overly complex with different flags.

Reply:
Not deprecating the `ENABLE_UPGRADERS` macro. But a simple way for users to revert to old `min_version_ = std::max(min_version_, get_min_version_for_kind(node->kind()));` behavior rather than relying solely on upgraders to compute version to write. `ENABLE_UPGRADERS` is difficult to toggle by an end user.